templates/includes/beta-banner.html

```diff
@@ -1,10 +1,14 @@
+{% load wagtailcore_tags %}
 <div class="beta-banner" aria-label="About The National Archives' beta">
-    <p class="beta-banner__standfirst">What does the 'beta' label mean?</p>
-    <p class="beta-banner__text">
-        This beta site contains new services and features currently in development. Many of these features are works in 
-        progress and are being updated regularly.  You can help us improve them by providing feedback as you use the site.
-    </p>
-    <div class="beta-banner__options">
-        <a class="beta-banner__link" href="/">Find out more</a>
-    </div>
+
+         <div class="beta-banner__standfirst">
+         {{ settings.core.SiteSettings.beta_banner_standfirst }}
+         </div>
+        <div class="beta-banner__text">
+            {{ settings.core.SiteSettings.beta_banner_text|richtext }}
+        </div>
+        <div class="beta-banner__options">
+            <a class="beta-banner__link" href="{{ settings.core.SiteSettings.beta_banner_link }}">Find out more</a>
```

Comment:
Could we move `{{ settings.core.SiteSettings.beta_banner_link }}` to the link text i.e. replace “Find out more”?

With regards to the `href`, it can stay as “/” for now because I’m not too sure where the link is going to go; it’s either going to go to a survey or the about page (as per a conversation I had with Lauren yesterday) and I need to confirm.

Reply:
@ababic - if the link takes the user to a third party website (as could be the case with the survey), we'll need to add `rel="noopener noreferrer"` and potentially `target="_blank"` to the `<a>` tag. Eventually, once the feedback mechanism has been completed, I believe the idea would be to always point to the About page thereafter. So I was thinking I could just implement the above to avoid creating additional potential backend complexity. But if it's straightforward to conditionally render the link attributes then happy to go with your suggestion.

Reply:
@JamesWChan it should be relatively simple to write a filter to detect if the URL goes to an external site, and conditionally add those bits in if so.